### ISSUE TITLE
Questions about ProteinBert n the global attention visualizing section

### ISSUE THREAD

**Issue by LTYRK** [nadavbra/protein_bert#13]
Hi  @nadavbra ,I have some questions about ProteinBert n the global attention visualizing section.
1. In the task of forecasting secondary structures, when I use
`  ('disorder_secondary_structure', OutputType(True, 'binary'))`
 the shape of finetuned_attention_values and pretrained_attention_values in 
`axes[2].barh(np.arange(seq_len), 100 * (finetuned_attention_values - pretrained_attention_values).sum(axis=0), color='#28B463') `
are different so they can't subtractible, but when I use (False, 'binary'), the shape of them are same. So how do I change this part of the code.

     
2. For global attention visualizing, when the task is about predict signal peptide, A sequence with a only one label, we can therefore easily analyse the effect of different amino acid global attention value on the whole sequence prediction, i.e. on the final predicted value. But in the task of predict secondary structures, Each amino acid site corresponds to a label, so how do I analyse the different global attention value in relation to these sites?

Thank you .

**Comment by nadavbra** [comment]
Hi @LTYRK 
1. What are the shapes of `finetuned_attention_values` and `pretrained_attention_values` that you obtain?
2. I'm not sure I understand your question. The global attention is, by definition, global, meaning that they don't correspond to any specific location. That's one of the main differences between ProteinBERT's architecture to that of other BERT-like models (which typically use self-attention, which is location-specific).
